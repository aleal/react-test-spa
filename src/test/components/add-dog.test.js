import React from 'react';
import {renderComponent} from '../test-helper';
import AddDog from '../../components/add-dog';

describe('Add Dog Tests' , () => {
    
    let component;

    beforeEach(() => {
        component = renderComponent(AddDog);
    });
  
    it('renders add dog page', () => {
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });
});