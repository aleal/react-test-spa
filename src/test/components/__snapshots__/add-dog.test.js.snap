// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Add Dog Tests renders add dog page 1`] = `
<div
  className="container"
>
  <h3>
    Add Dog
  </h3>
  <div
    className="form"
  >
    <form
      onSubmit={[Function]}
    >
      <div
        className=""
      >
        <div
          className="text-help"
        >
          
        </div>
      </div>
      <div
        className="form-group "
      >
        <label>
          Name
        </label>
        <input
          className="form-control"
          name="name"
          onBlur={[Function]}
          onChange={[Function]}
          onDragStart={[Function]}
          onDrop={[Function]}
          onFocus={[Function]}
          type="text"
          value=""
        />
        <div
          className="text-help"
        >
          
        </div>
      </div>
      <div
        className="form-group "
      >
        <label>
          Picture
        </label>
        <input
          className="form-control"
          name="picture"
          onBlur={[Function]}
          onChange={[Function]}
          onDragStart={[Function]}
          onDrop={[Function]}
          onFocus={[Function]}
          type="file"
        />
        <div
          className="text-help"
        >
          
        </div>
      </div>
      <button
        className="btn btn-primary"
        type="submit"
      >
        Submit
      </button>
      <a
        className="btn btn-danger"
        href="/home"
        onClick={[Function]}
      >
        Cancel
      </a>
    </form>
  </div>
</div>
`;
